.github/ISSUE_TEMPLATE/developer-onboarding.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Developer Onboarding
+about: Onboarding steps for developers.
+title: 'Developer Onboarding: GH_HANDLE'
+labels: dev, onboarding
+assignees: loganmeetsworld
+
+---
+
 # Developer Onboarding
 
 Onboardee: _GH handle of person being onboarded_
```

Comment:
Should these have a "-" in front to put them on separate lines as bullet points?